components.json

```
@@ -457,6 +457,23 @@
 			"title": "Git",
 			"owner": "lgiraudel"
 		},
+		"glimmer": {
+			"title": "Glimmer / Ember",
+			"require": "markup-templating",
```

You also directly use `markup` in your language definition.

I know that `markup-templating` requires `markup`, so `markup` is transitively required but it's better to be explicit.

```suggestion
			"require": ["markup", "markup-templating"],
```